docs/launch.md

```diff
@@ -46,16 +46,57 @@ In this diagram the Activity Player could be substituted for another third party
   - **state** this is a random string that is used as a key to restore all of the params from the launch after OAuth2 is done. The state has a size limit so the params themselves cannot be sent directly as the state.
 - Code Location: `api.ts#authorizeInPortal`
 
+#### Notes on URLs generated by portal for portal-report
+
+When the portal redirects back to the portal report it uses a URL like:
+https://portal-report.concord.org/branch/master/index.html#access_token=231cb8cfd58438f234c1c0b57e2d9053&token_type=bearer&expires_in=604800&state=2bf6oio5yg1
+
+The `access_token` in this url is intended for the portal-report to get a JWT that it then will use to make further requests. However currently the `access_token` is used for the offering and classes requests too:
+https://learn.concord.org/api/v1/offerings/135999
+https://learn.concord.org/api/v1/classes/60888
+
+Here is an example request for the JWT:
+https://learn.concord.org/api/v1/jwt/firebase?firebase_app=report-service-pro&class_hash=7883841115415ce2255ec7d0eef80f8c6e096191d63b5be1&resource_link_id=135999&target_user_id=1063089
+
+Note that the `target_user_id` parameter is passed. This is necessary so the returned JWT will include the `target_user_id` property if the researcher requesting the JWT has access to this target user's data.
+
+##### To do: Researchers, project researchers and project admins should be able to load interactive links from the details report
+
+The details report generated by report-service includes links to view the interactives with students' saved state. These links can be opened by portal wide admins or teachers in the class. The problem is that these links cannot currently be opened by portal wide admins, project researchers, or project admins.
```

Comment:
I think this has a typo in it. I think portal wide admins can open these links.

Reply:
Gotcha. Portal-wide admins can open the links. I think it's just project researchers and project admins (although I'm not sure what a project admin is, actually).